datahub-web-react/src/app/analytics/event.ts

```diff
@@ -64,6 +64,8 @@ export enum EventType {
     SelectAutoCompleteOption,
     SelectQuickFilterEvent,
     DeselectQuickFilterEvent,
+    ChromeExtensionProfileViewEvent,
```

Comment:
Minor: What if we name these EmbedProfileViewEvent?

Reply:
yup that's better! i was so stuck in the naming from chrome extension purposes